config.go

```diff
@@ -30,18 +32,22 @@ const defaultConfigFile = "config.yaml"
 func defaultConfig() Config {
 	return Config{
 		HTTPPort: 1305,
+		BaseURL:  "",
 		Node: Node{
 			Address: "http://localhost:8081",
 		},
 		AccessLogs: true,
+		OIDC:       oidc.DefaultConfig(),
 	}
 }
 
 type Config struct {
-	HTTPPort   int  `koanf:"port"`
-	Node       Node `koanf:"node"`
-	AccessLogs bool `koanf:"accesslogs"`
+	HTTPPort   int    `koanf:"port"`
+	BaseURL    string `koanf:"base_url"`
```

Comment:
With koanf config library, when using environment variables, underscores are used for nested structs, meaning you can't set `base_url` through the environment variables. So you need to remove the underscore or come up with a different name. What about just "url"?